docs/libcurl/opts/CURLOPT_CAPATH.3

```diff
@@ -60,8 +60,8 @@ if(curl) {
 }
 .fi
 .SH AVAILABILITY
-This option is supported by the OpenSSL, GnuTLS and mbedTLS (since 7.56.0)
-backends.
+This option is supported by the OpenSSL, GnuTLS, mbedTLS (since 7.56.0) and
+wolfSSL (since 8.4.0) backends.
```

Comment:
I don't think this is correct. The code already uses this argument before this change. Since at least 7.65.2 (deb9462ff)

Reply:
I think the feature check came later, which then made it not work again - unless set in configure.

Reply:
Right, but it was not exposed through CLI, so it was incomplete. I will drop this hunk. Good catch.

Reply:
Do you want to address this doc fix in your PR as well?

Reply:
I have dropped this hunk.